app/components/navigation/SidebarNavigation.tsx

```diff
@@ -6,7 +6,7 @@ export default function SidebarNavigation(props: {
   actions: ReactNode;
 }) {
   return (
-    <div className="pt-24 pb-64">
+    <div id="sidebar-navigation-content" className="pt-24 pb-64">
```

Comment:
this is required since there are `identification` button available in top nav and side bar nav